### PR TITLE
feat: Create MyApps Page

### PR DESCRIPTION
**Pull request by yashpandit**
# Pull Request Template

## Description

Create myapps page.

Resolves: #39 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Unit Test
- [ ] Integration Test

## Checklist

- [x] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by yashpandit** [comment]
my-apps-1

File: client/src/components/common/index.js [hunk 1]
```diff
@@ -3,8 +3,8 @@ import Select from './Select';
 import Heading from './Heading';
 import LabelItem from './LabelItem';
 import RowCol from './RowCol';
-import InfoCard from './InfoCard';
+import CardWithImage from './CardWithImage';
 
 export {
```
Comment (nimish-gupta) [inline client/src/components/common/index.js]: You can remove this export and directly export the files. Check this answer, https://stackoverflow.com/a/35665938

File: client/src/components/common/InfoCard.js [hunk 2]
```diff
@@ -2,17 +2,22 @@ import React from 'react';
 import {
   Card, CardBody, CardText, CardTitle,
 } from 'reactstrap';
+import PropTypes from 'prop-types';
 
-const InfoCard = () => {
+const InfoCard = (props) => {
```
Comment (nimish-gupta) [inline client/src/components/common/InfoCard.js]: You can destructure the props here only if you are not passing the entire/ spreading props in any other component.

File: client/src/components/common/CardWithImage.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Col } from 'reactstrap';
+import PropTypes from 'prop-types';
+import InfoCard from './InfoCard';
+import Image from './Image';
+
+const CardWithImage = (props) => {
```
Comment (nimish-gupta) [inline client/src/components/common/CardWithImage.js]: You can destructure the props here only if you are not passing the entire/ spreading props in any other component.

File: client/src/components/common/index.js (hunk 1 above)
Comment (yashpandit) [reply client/src/components/common/index.js]: Will make the changes. Thanks!

File: client/src/components/common/CardWithImage.js (hunk 3 above)
Comment (yashpandit) [reply client/src/components/common/CardWithImage.js]: Cool. Will keep this in mind from now on. Thanks!

**Pull request merged**